providers/dns/cloudxns/client_test.go

```diff
@@ -0,0 +1,283 @@
+package cloudxns
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func handlerMock(method string, response *apiResponse, data interface{}) http.Handler {
+	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		if req.Method != method {
+			content, err := json.Marshal(apiResponse{
+				Code:    666,
```

Comment:
666? Needs comments as to why this is the code.

Reply:
It's just a dumb code